sale_payment_method/i18n/de.po

```diff
@@ -0,0 +1,157 @@
+# Translation of OpenERP Server.
+# This file contains the translation of the following modules:
+#	* sale_payment_method
+#
+# Rudolf Schnapka <schnapkar@golive-saar.de>, 2015.
+msgid ""
+msgstr ""
+"Project-Id-Version: OpenERP Server 7.0\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2013-05-02 07:06+0000\n"
+"PO-Revision-Date: 2015-01-02 16:46+0100\n"
+"Last-Translator: Rudolf Schnapka <schnapkar@golive-saar.de>\n"
+"Language-Team: German <kde-i18n-de@kde.org>\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
+"Language: de\n"
+"X-Generator: Lokalize 1.5\n"
+
+#. module: sale_payment_method
+#: view:sale.order:0
+msgid "Automation Information"
+msgstr "Information zum Automatismus"
+
+#. module: sale_payment_method
+#: help:payment.method,journal_id:0
+msgid ""
+"If a journal a selected, when a payment is recorded on a backend, payment "
+"entries will be created in this journal. "
+msgstr ""
+"Wenn ein Journal ausgewählt ist, wird bei einem im Backend verbuchtem "
+"Zahlungseingang, dieser auch in diesem Journal eingetragen."
+
+#. module: sale_payment_method
+#: view:payment.method:0
+msgid ""
+"Even if the E-commerce backend receives a payment for a sale order,\n"
+"                                no payment entries will be generated."
+msgstr ""
+"Auch wenn im E-Commerce-Backend ein Zahlungseingang verbucht wird,\n"
+"                 werden hier keine Buchungen zu Zahlungen angelegt."
```

Comment:
This in an error in the string due to the use of triple quotas, but don't reply it on your translation. Remove \n and the extra spaces